src/components/ControlBar/ViewControlBar/styles/DashboardsBar.module.css

```diff
@@ -0,0 +1,112 @@
+.container {
+    position: relative;
+    background-color: var(--colors-white);
+    box-shadow: rgba(0, 0, 0, 0.2) 0px 0px 6px 3px;
+    z-index: 2001;
+    overflow: hidden;
+    box-sizing: border-box;
+    flex: none;
+    display: flex;
+    flex-direction: column;
+}
+
+.content {
+    padding: 10px 6px 0 6px;
+    overflow: hidden;
+    margin-bottom: 21px; /* to make space for the show more button */
+}
+
+.expanded .content {
+    overflow-y: auto;
+}
+
+.one {
+    height: 71px;
+}
+.two {
+    height: 111px;
+}
+.three {
+    height: 151px;
+}
+.four {
+    height: 191px;
+}
+.five {
+    height: 231px;
+}
+.six {
+    height: 271px;
+}
+.seven {
+    height: 311px;
+}
+.eight {
+    height: 351px;
+}
+.nine {
+    height: 391px;
+}
+.ten {
+    height: 431px;
+}
+
+.expanded .container {
+    height: 431px;
+}
+
+.topMargin {
```

Comment:
Is this the best name for this class?

Reply:
How about `spacer`, or `topOffset` ?

Reply:
sure, `spacer` sounds good to me and can't be confused for a css prop 👍